Removed the -s from the install command. Without this, usb_modeswitch would
fail to install.

Signed-off-by: J.C. Woltz <jwoltz@gmail.com>

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -46,7 +46,7 @@
 	cd jim && $(MAKE) distclean
 
 install-common: all
-	install -D -s --mode=755 usb_modeswitch $(SBINDIR)/usb_modeswitch
+	install -D --mode=755 usb_modeswitch $(SBINDIR)/usb_modeswitch
 	install -D --mode=755 usb_modeswitch.sh $(UDEVDIR)/usb_modeswitch
 	install -D --mode=644 usb_modeswitch.conf $(ETCDIR)/usb_modeswitch.conf
 	install -D --mode=644 usb_modeswitch.1 $(MANDIR)/usb_modeswitch.1
@@ -67,10 +67,10 @@
 install: install-common install-script
 
 install-shared: dispatcher-dynamic install-common
-	install -D -s --mode=755 usb_modeswitch_dispatcher $(SBINDIR)/usb_modeswitch_dispatcher
+	install -D --mode=755 usb_modeswitch_dispatcher $(SBINDIR)/usb_modeswitch_dispatcher
 
 install-static: dispatcher-static install-common
-	install -D -s --mode=755 usb_modeswitch_dispatcher $(SBINDIR)/usb_modeswitch_dispatcher
+	install -D --mode=755 usb_modeswitch_dispatcher $(SBINDIR)/usb_modeswitch_dispatcher
 
 uninstall:
 	$(RM) $(SBINDIR)/usb_modeswitch
